functions/index.js

```diff
@@ -37,6 +37,10 @@ const signupUser = (req, res) => {
         password: req.body.password,
         confirmPassword: req.body.confirmPassword,
         username: req.body.username,
+        firstName: req.body.firstName,
+        lastName: req.body.lastName,
+        location: req.body.location,
+        lastSeen: req.body.lastSeen,
```

Comment:
thanks for this. I think the lastSeen field shouldn't be sent by the client. We should calculate it in the server and then assign the value, for example in JavaScript is something like this:

```
const newUser = {
    [other stuff],
    lastSeen: new Date().toISOString(),
};
```

With that you just created a new timestamp for when the user was created. Pretty cool right?